projects.html

```diff
@@ -20,5 +20,6 @@
         <p><a href="https://github.com/CHG-MERIDIAN/CHG.Extensions.Security.Txt">A middleware to represent the "security.txt" for ASP.NET Core applications.</a> by <a href="https://github.com/CHG-MERIDIAN">CHG-MERIDIAN AG</a>.</p>
         <p><a href="https://github.com/HarmlessSystems/security.txt">A browser extension for discovering security.txt files</a> by <a href="https://harmless.systems/">Harmless Systems</a>.</p>
         <p><a href="https://github.com/Vacilando/gatsby-plugin-security-txt">A configurable security.txt plugin for GatsbyJS</a> by <a href="https://www.linkedin.com/in/vacilando/">Tomas Fülöpp</a>.</p>
-    </div>
+        <p><a href="https://github.com/disclose/diostst">A Go scraper that validates security.txt files and outputs them in the disclose.io JSON format.</a> by <a href="https://disclose.io">disclose.io</a>.</p>
```

Comment:
Thanks, Casey! I think the link is missing an `s` on the end. Once that has been added, I will merge this PR. :)

```suggestion
        <p><a href="https://github.com/disclose/dioststs">A Go scraper that validates security.txt files and outputs them in the disclose.io JSON format.</a> by <a href="https://disclose.io">disclose.io</a>.</p>
```

Reply:
Hey @EdOverflow: Updated. Thanks!